app/services/backup_service.rb

```diff
@@ -142,5 +142,7 @@ def download_to_tar(tar, attachment, filename)
         io.write(buffer)
       end
     end
+  rescue Errno::ENOENT
+    Rails.logger.warn "Error making a backup for attachment #{attachment}"
```

Comment:
Not sure if the string output of `attachment` is useful information..

Reply:
Hm, changed to display the filename (which really is the file path) instead